client/my-sites/activity/activity-log-v2/index.tsx

```diff
@@ -95,9 +95,10 @@ const ActivityLogV2: FunctionComponent = () => {
 				jetpackCloudHeader
 			) : (
 				<FormattedHeader
+					brandFont
 					headerText="Activity"
 					subHeaderText={ translate(
-						'This is the complete event history for your site. Filter by date range and/or activity type.'
+						"Keep tabs on all your site's activity — plugin and theme updates, user logins, setting modifications, and more."
```

Comment:
Should this be `setting modifications` or `settings modifications`?

diff
```diff
- setting modifications
+ settings modifications
```

Reply:
I'm no copy expert, but I think both forms could work. Personally my brain gravitates more toward the plural "settings modifications," but I think either could be grammatically correct. It's worth noting, this copy is pulled directly from the original Calypso Blue `ActivityLog` component, so it's already got translations on file.